### DISCUSSION TITLE
WinEdt Discussion Forum:   Read Me

### DISCUSSION THREAD

**Discussion by anton-vrba** [WinEdt-Team/WinEdt-Support#5] [Read Me]
<h1><span><img src="https://github.com/WinEdt-Team/WinEdt-Support/assets/34773801/0d1caf2a-d7e1-495d-be67-7035f28781b7" width="60" height="40" style="vertical-align:bottom;margin:0px 0px">  WinEdt <a href="https://github.com/WinEdt-Team/WinEdt-Support/discussions" > discussions </a> </span></h1>

Welcome to WinEdt discussion to support the WinEdt editor which can be downloaded from our [home page](https://www.winedt.com/). New users should first read the WinEdt [QuickGuide.pdf](https://www.winedt.com/doc/QuickGuide.pdf).

Please acquaint yourself with our discussion forum which has following [structure](https://github.com/WinEdt-Team/WinEdt-Support/discussions/4).  All new discussions are initialized using the GitHub web pages as [outlined here](https://github.com/WinEdt-Team/WinEdt-Support/discussions/13), however discussion can be replied by email; default email notifications are available to the user and the WinEdt-Team.  All users have the ability to [obtain email notifications. ](https://github.com/WinEdt-Team/WinEdt-support/discussions/3)

[Alex Simonic](https://www.winedt.com/) will continue to give expert technical advice regarding the ins and outs of WinEdt. [Anton Vrba](https://github.com/anton-vrba) will advise on matters pertaing to the GitHub dicussion forum. To participate in these discussions you need a GitHub account; should you have problems registering with GitHub contact Anton -- email: alv (at) iow.onl

**Comment by Alex-Simonic** [comment]
Please do not post your question as a comment or a reply to an unrelated posting. Start a new discussion with an informative title instead. This way postings get properly archived. 

We will tolerate errors like this but out-of-place postings will not be answered and will be removed from the repository. Post again as a new discussion with an informative title!
